sui/sources/date.move

```diff
@@ -30,10 +28,10 @@ module movemate::date {
     const DOW_SAT: u64 = 6;
     const DOW_SUN: u64 = 7;
 
-    const EYEAR_BEFORE_1970: u64 = 0;
-    const EADDITION_ASSERTION_FAILED: u64 = 1;
-    const ESUBTRACTION_ASSERTION_FAILED: u64 = 2;
-    const EFROM_TIMESTAMP_LATER_THAN_TO_TIMESTAMP: u64 = 3;
+    const EYEAR_BEFORE_1970: u64 = 0x10000;
+    const EADDITION_ASSERTION_FAILED: u64 = 0xB0001;
+    const ESUBTRACTION_ASSERTION_FAILED: u64 = 0xB0002;
+    const EFROM_TIMESTAMP_LATER_THAN_TO_TIMESTAMP: u64 = 0xB0003;
```

Comment:
Is there any reason why we jump from:
`0x10000` -> `0xB0001` -> `0xB0002` -> `0xB0003`

instead of:
`0x10000` -> `0x10001` -> `0x10002` -> `0x10003`

or is it a typo?